.github/workflows/zizmor.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         with:
           persist-credentials: false
       - name: Install the latest version of uv
-        uses: astral-sh/setup-uv@887a942a15af3a7626099df99e897a18d9e5ab3a # v4
+        uses: astral-sh/setup-uv@4db96194c378173c656ce18a155ffc14a9fc4355 # v4
```

Comment:
Well that's annoying, that it doesn't update the comment. Is there any way to configure Dependabot to do so?

Reply:
Aha, it did in #1699, the problem is just that the comment didn't match the tag, so it didn't see it as needing updating. We should fix the tag to re-enable this.